sway-lib-std/src/bytes.sw

```diff
@@ -888,19 +906,19 @@ fn test_bytes_limits() {
 }
 
 #[test()]
-fn test_split() {
+fn test_split_at() {
```

Comment:
Can you add another test or two for edge cases where `mid` is zero and `mid` is `self.len`?